great_expectations/rule_based_profiler/domain_builder/table_domain_builder.py

```diff
@@ -1,11 +1,30 @@
 from typing import List, Optional, Union
 
+from great_expectations.core.batch import Batch, BatchRequest, RuntimeBatchRequest
 from great_expectations.execution_engine.execution_engine import MetricDomainTypes
 from great_expectations.rule_based_profiler.domain_builder import DomainBuilder
 from great_expectations.rule_based_profiler.types import Domain, ParameterContainer
 
 
 class TableDomainBuilder(DomainBuilder):
+    def __init__(
+        self,
+        data_context: "DataContext",  # noqa: F821
+        batch_list: Optional[List[Batch]] = None,
+        batch_request: Optional[Union[BatchRequest, RuntimeBatchRequest, dict]] = None,
+    ):
+        """
+        Args:
+            data_context: DataContext
+            batch_list: explicitly specified Batch objects foruse in DomainBuilder
```

Comment:
"for use"

Reply:
@NathanFarmer thank you! Fixing in next commit.